src/models/searchservice/request/Limit.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * The maximum limit of results per vertical.
+ * Each limit can be set from 1-50 inclusive.
+ *
+ * @public
+ */
+export interface Limit {
```

Comment:
After thinking about this a little more, leaving it as Limit might be fine because the API doesn't use different names for vertical vs unviersal limits